extensions/knative/annotator/pom.xml

```diff
@@ -53,6 +53,12 @@
         <dependency>
             <groupId>io.sundr</groupId>
             <artifactId>builder-annotations</artifactId>
+          <exclusions>
```

Comment:
How is this related to the TemplateInstance support?

Reply:
It was giving dependency issues for com.sun.tools:1.7 said that it's not available so just excluded it

Reply:
Weird, is your JAVA_HOME pointing to a JDK? Usually it fails when pointing to a JRE

Reply:
It's pointing to a JDK. This wasn't why the build was failing, it was just giving a warning